src/LocalHostStats.cpp

```diff
@@ -197,6 +215,25 @@ void LocalHostStats::luaHostBehaviour(lua_State* vm) {
 
 /* *************************************** */
 
+
+
+void LocalHostStats::luaDomainNamesBehaviour(lua_State* vm) {
```

Comment:
This `LocalHostStats::luaDomainNamesBehaviour` is an identical cut-and-paste of method above `LocalHostStats::luaHostBehaviour`. I believe this can be solved by using class `Cardinality` as suggested in the other review comment.